### PR TITLE
Add Extension Build Scripts and Readme file

### LINKED ISSUE #8: Add CRX makefile
We need a makefile that produces a crx3 file for the extension.

### PR DESCRIPTION
**Pull request by michaelehab**
Fix #8 
Added shell and bat scripts that use an installed Chrome based browser (by default Google Chrome) to build the vulntotal extension CRX file.

### CODE REVIEW

**Comment by michaelehab** [comment]
@keshav-space @OmkarPh Can we check this?

**Comment by OmkarPh** [comment]
Hi michael, 
for macos, the "chrome" command doesn't work out of the box, can you update the `release.sh` to -

```
# Variables
PEM_FILE="dist/vulntotal-extension.pem"
OUTPUT="dist/vulntotal-extension.crx"
SRC_DIR="$(pwd)/src"

if [[ "$OSTYPE" == "darwin"* ]]; then
    CHROME_BASED_BROWSER="/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
else
    CHROME_BASED_BROWSER="chrome"
fi

mkdir -p "dist"

echo "Packing the vulntotal extension into a .crx file..."
"$CHROME_BASED_BROWSER" --pack-extension="$SRC_DIR"

...
```

Also, check once if it works fine in linux also after this change

**Comment by michaelehab** [comment]
Updated it and tested it on linux, everything is working as expected.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+```sh
+./release.sh
+```
+- On Windows, run:
+```sh
+release.bat
+```
+This will generate the extension file (.crx) in the dist folder.
+
+## Load the Extension:
```
Comment (keshav-space) [inline README.md:27]: Also add instruction to load extension in firefox.

**Review by keshav-space — APPROVED** [review]
Thanks @michaelehab, please add the missing instruction to load extension on firefox.

File: README.md (hunk 1 above)
Comment (michaelehab) [reply README.md:27]: We can't install crx extensions manually on firefox, I believe we'll add a section later for download links when it gets published on extensions stores for chrome, firefox, edge..etc

**Pull request merged**